### ISSUE TITLE
Build errors

### ISSUE THREAD

**Issue by steveb999** [aoool/cloudtiering#20]
Sergey,

Thankyou for you detailed email reply. My libs3 seem to be the version you suggest.
```
steve@steve:~/CloudTieringFS$ ls -l /usr/lib/libs3.so*
lrwxrwxrwx 1 root root    10 Sep 27  2017 /usr/lib/libs3.so -> libs3.so.2
lrwxrwxrwx 1 root root    12 Sep 27  2017 /usr/lib/libs3.so.2 -> libs3.so.2.0
-rw-r--r-- 1 root root 72344 Sep 27  2017 /usr/lib/libs3.so.2.0
steve@steve:~/CloudTieringFS$ 
```

---
Original email conversation is in the attachment.
[Inbox _ sergey@morozov.ch _ ProtonMail.pdf](https://github.com/ser94mor/CloudTieringFS/files/3411610/Inbox._.sergey%40morozov.ch._.ProtonMail.pdf)

**Comment by ser94mor** [comment]
I will investigate further. Could you please specify the distribution you are using and whether or not you have `libs3-devel` installed? Look into the /usr/include/... . It might be that, in case you are using Ubuntu or Debian, there is no `libs3-devel` package available in the repositories.

**Comment by steveb999** [comment]
Im on ubuntu 19.04. Im assuming this is the same just different naming convention.
libs3-dev is already the newest version (2.0-3)

**Comment by ser94mor** [comment]
I will try Ubuntu 19.04 on a virtual machine and will get back to you.

**Comment by steveb999** [comment]
libattr1-dev libdotconf-dev libs3-dev dotconf-dev
Thanks

**Comment by ser94mor** [comment]
@steveb999, I was able to compile the code on Ubuntu 19.04. It required some changes to Makefile and the code. It was not easy. I will share a branch with these changes soon, so that you can use it. Preparation will take some time.

**Comment by ser94mor** [comment]
@steveb999, please, check `fix-ubuntu-19.04` branch. It should work with the `libs3` and `dotconf` versions available in the Ubuntu 19.04 repositories.

https://github.com/ser94mor/CloudTieringFS/tree/fix-ubuntu-19.04

**Comment by ser94mor** [comment]
Closing this issue since Ubuntu 19.04-specific build errors have been fixed in branch `fix-ubuntu-19.04`. @steveb999 confirmed it via the email.
